app/code/Inchoo/Sample05/ViewModel/Event.php

```diff
@@ -85,14 +103,23 @@ public function getEventById(int $id): ?EventInterface
      * @param int|null $pageSize
      * @return EventInterface[]
      */
-    public function getEventList(int $pageSize = null): array
+    public function getEventList(int $pageSize = 5): array
     {
         if ($pageSize) {
             $this->searchCriteriaBuilder->setPageSize($pageSize);
         }
 
         $searchCriteria = $this->searchCriteriaBuilder->create();
 
+        $this->filter
+            ->setField('status')
+            ->setValue('1')
+            ->setConditionType('eq');
+
+        $this->filterGroup->setFilters([$this->filter]);
+
+        $searchCriteria->setFilterGroups([$this->filterGroup]);
+
```

Comment:
Nisi morao injectati Filter i FilterGroup objeckte, imaš metodu na SearchCriteriaBuilderu

```suggestion
        $this->searchCriteriaBuilder->addFilter('status', 1);
```